src/components/Dashboard/FilingHistoryList/CompletedIa.vue

```diff
@@ -29,22 +30,7 @@ export default class CompletedIa extends Vue {
 </script>
 
 <style lang="scss" scoped>
-.to-dashboard-container {
-  text-align: center;
-  padding: 1rem;
-}
-
-h4 {
-  letter-spacing: 0;
-  font-size: 0.9375rem;
-  font-weight: 700;
-}
-
-p:first-of-type {
-  padding-top: 0.75rem;
-}
-
 p {
-  margin-bottom: 0.5rem !important;
+  margin-top: 1rem !important;
```

Comment:
All detail component paragraphs now have top and bottom margin as 1rem.

Note: margins are used for spacing outside the element (where padding is for spacing inside the element). In this case, we want whitespace  between elements, so margins make sense. Also, margins can overlap, which simplifies things.